Sources/Plasma/Apps/plPythonPack/main.cpp

```diff
@@ -213,18 +213,19 @@ void FindFiles(std::vector<plFileName> &filenames, std::vector<plFileName> &path
     // Get the names of all the python files
     std::vector<plFileName> pys = plFileSystem::ListDir(path, "*.py");
 
-    for (auto iter = pys.begin(); iter != pys.end(); ++iter)
+    for (const auto& py : pys)
     {
-        filenames.push_back(iter->GetFileName());
+        filenames.push_back(py.GetFileName());
```

Comment:
```suggestion
        filenames.emplace_back(py.GetFileName());
```

While we're here, let's make this more technically correct.